openpype/modules/deadline/plugins/publish/validate_expected_and_rendered_files.py

```diff
@@ -21,7 +21,12 @@ class ValidateExpectedFiles(pyblish.api.InstancePlugin):
 
     def process(self, instance):
         self.instance = instance
-        frame_list = self._get_frame_list(instance.data["render_job_id"])
+
+        if "gather" in instance.data["name"]:
+            frame_range = range(1001, int(instance.data["frameEndHandle"]))
```

Comment:
this is hardwired at 1001, but we need to actually get the frame number from the first and last frame of the repre. even frameEndHandle is not always present. it's probably best to take the first and last items from repre["files"] if it's a list and regex to find a number sequence from the end of the file string. If instead this is a mov (so repre["files"] is just a string we can fallback to frameStartHandle and frameEndHandle, and if they are not in the instance we need to do (frameEnd + handleEnd) - (frameStart - handleStart)